nbgrader/docs/source/user_guide/third_party.rst

```diff
@@ -50,6 +50,9 @@ Libraries, Tools, and Platforms
 Presentations and Educational Materials
 ---------------------------------------
 
+`Teaching and Learning with Jupyter Book <https://jupyter4edu.github.io/jupyter-edu-book/>`_
```

Comment:
Hmm, looks like this snuck in here because I think you merged master into your branch. I am going to merge master into the `multiple-classes-dev` branch and then you'll need to do another merge (or rebase) from `multiple-classes-dev` into your branch so these unrelated changes get dropped.

Reply:
Actually, I suspect that's going to cause a lot of merge conflicts, so I will actually leave it as is for now and do the merge later. It's ok if these changes sneak in for now :)